### PR TITLE
rox-filer: init at 2.11

### PR DESCRIPTION
**Pull request by proteansec**
###### Motivation for this change

Addition of rox-filer package manager.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by joachifm** [comment]
See also https://github.com/NixOS/nixpkgs/pull/17650

File: pkgs/desktops/rox/rox-filer/default.nix [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    ln -sv application-x-{bzip-compressed-tar,lzo}.png
+    ln -sv application-x-{bzip,xz}.png
+    ln -sv application-x-{gzip,lzma}.png
+    ln -sv application-{msword,rtf}.png
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Fast, lightweight, gtk2 file manager";
+    homepage = "http://rox.sourceforge.net/desktop";
+    license = "GPL-2 LGPL-2";
```
Comment (joachifm) [inline pkgs/desktops/rox/rox-filer/default.nix]: Please use `stdenv.lib.licenses.*`

**Comment by proteansec** [comment]
@joachifm fixed

**Comment by joachifm** [comment]
@proteansec the other implementation had an issue with trying to download xml schema during the build; a cursory look at this version suggests that will be a problem here, too (network access is disallowed within the build sandbox).

**Comment by proteansec** [comment]
@joachifm I'm not sure, I don't have any such problems although I'm using **nix.useSandbox** on NixOS. In any case, it would be best if we merge the two pull requests.

**Comment by joachifm** [comment]
@proteansec your version doesn't build the rox manual, that'd account for the difference. I like your approach to dealing with appdir better than the other version. Given that it also builds in a sandboxed environment, I'd be inclined to integrate this PR.

**Comment by Mic92** [comment]
see also: https://github.com/NixOS/nixpkgs/pull/17650

**Pull request merged**